### ISSUE TITLE
Working sample of AWS::ServiceCatalog::CloudformationProduct is required

### ISSUE THREAD

**Issue by jarvisdreams9** [awsdocs/aws-cloudformation-user-guide#119]
Current Documentation:

Info The URL of the CloudFormation template in Amazon S3. Specify the URL in JSON format as follows:
"LoadTemplateFromURL": "https://s3.amazonaws.com/cf-templates-ozkq9d3hgiq2-us-east-1/..."

The above is confusing as we do not know if we it needs to be a JSON string or a JSON object. More importantly, it throws an "Internal Failure" when used the wrong way.

Suggested:

```JSON
                "ProvisioningArtifactParameters": [
                    {
                        "Info": {
                          "LoadTemplateFromURL": "https://s3-<region>.amazonaws.com/..."
                        }
                    }
                ]
```

**Comment by AdamsNimbus** [comment]
I am having issues supplying this in a YAML Cloudformation template... Very misleading.

How did you solve it?

**Comment by blade2005** [comment]
Thank you @jarvisdreams9 for raising this issue. Because of this I was able to correct my cft.

**Comment by tandonsw** [comment]
Here's a YAML sample:

      ProvisioningArtifactParameters: 
        - 
          Info: { "LoadTemplateFromURL": "https://s3.us-east-2.amazonaws.com/..." }

**Comment by mdaehnert** [comment]
Update on the YAML example of @tandonsw 

When using YAML, there is no need to use JSON inside. Instead just create the elements like this:
```yml
  ProvisioningArtifactParameters: 
  - Info:
      LoadTemplateFromURL: https://s3.us-east-2.amazonaws.com/...
```

**Comment by mdaehnert** [comment]
I also struggled on the JSON value and run into the Internal Error.

So type JSON is misleading. Please let's use a different Sub-Documentation-Page for the new property like: _AWS::ServiceCatalog::CloudFormationProduct ProvisioningArtifactProperties Info_

**Comment by mjohnson159** [comment]
yep, this is super ambiguous to me. Am I forced to have the source template be in json format to be able to reference here? I write all my templates in yaml as its easier and more readable for me, so I'd had to have to convert certain templates to json just so I can reference them via this template to automate adding them to Service Catalog when I create a new template. Forcing each new template to be written in json to use in the `AWS::ServiceCatalog::CloudFormationProduct ProvisioningArtifactProperties Info` field is really not ideal.

@laurhans is it possible this can be looked at again? I'm sure its really low priority, but its really mucking with a large scale Service Catalog deployment pipeline I'm trying to set up for a customer as its causing me to have to explain which templates can be in either yaml or json and which _have_ to be in json format.

**Comment by snivas** [comment]
I would like to point one more,

As per the documentation one can use any of S3 URL,

> The URL of the AWS CloudFormation template in Amazon S3, AWS CodeCommit, or GitHub in JSON format. Specify the URL in JSON format as follows:

But there is no example on how to use CodeCommit URL. I have tried with browse URL of template but it doesn't work. Better documentation on this would be helpful

**Comment by joshbean** [comment]
Closing this issue or pull request in advance of archiving this repo. For more information about the decision to archive this repo (and others in the "awsdocs" org), see [the announcement on the AWS News Blog](https://aws.amazon.com/blogs/aws/retiring-the-aws-documentation-on-github/).
